test/e2e/tests/dapp-interactions/revoke-permissions.spec.js

```diff
@@ -43,4 +43,47 @@ describe('Wallet Revoke Permissions', function () {
       },
     );
   });
+
+  it('should revoke "endowment:permitted-chains" permissions', async function () {
```

Comment:
Nit: Nice to see us add coverage for this, but perhaps this can be made in a separate PR? Looks like the behavior it's covering is pre-existing

Reply:
this behavior is broken in `main` though, as in `endowment:permitted-chains` can be revoked independently of `eth_accounts`

Reply:
Revoking `permitted-chains` still works on `main` though. We'd need to update the assertion if we port this to `main`, but that's about it I think